app/views/list_responses/feedbacks.html.slim

```diff
@@ -0,0 +1,66 @@
+= render 'reminder_mailings/section_navigation'
+
+h1 Eingereichte Feedbacks
+
+nav.navbar.section-navigation.hidden-print
+  ul.list-inline
+    li= button_link 'Filter aufheben', url_for(q: { s: ['updated_at asc'] }), dimension: :sm
+    = custom_filter_dropdown('Geprüft',
+        { q: :reviewer_id_not_null, text: 'Angeschaut', value: 'true' },
+        { q: :reviewer_id_null, text: 'Ungesehen', value: 'true' })
+    = custom_filter_dropdown('Autor',
+        { q: :author_volunteer, text: 'Freiwillige/r', value: 'true' },
+        { q: :author_volunteer, text: 'AOZ', value: 'false' })
+
+= bootstrap_paginate(@feedbacks)
+
+table.table.table-striped.list-responses-table
+  thead
+    tr
+      th.limit-width= sort_link @q, :volunteer_contact_full_name, 'Freiwillige/r'
+      th.limit-width Einsatz
+      th= sort_link @q, :author_profile_contact_full_name, 'Autor/in'
+      th Ziele
+      th Erfolge
+      th Zukunft
+      th= t_attr(:comments, Feedback)
+      th.limit-width= sort_link @q, :conversation, 'Gespräch'
+      th= sort_link @q, :created_at, 'Alter'
+      th.limit-width
+
+  tbody
+    - @feedbacks.each do |record|
+      tr*{ data: { feedback_id: record.id } }
+        td.index-action-cell
+          a href=volunteer_path(record.volunteer)
+            span #{record.volunteer.contact.last_name},
+            span= record.volunteer.contact.first_name
+        td.index-action-cell
+          a href="#{url_for(record.feedbackable)}" title="#{record.feedbackable.to_label}"
+            - record.feedbackable.label_parts.reject(&:blank?).each do |label_part|
+              span= label_part.truncate(30)
+        td
+          - if record.author.volunteer?
+            = link_to record.volunteer.freiwillig_gendered, record.volunteer
+          - else
+            = link_to record.author.profile.contact.natural_name, record.author
+        td= record.goals
+        = td_truncate_content_modal(record.achievements, 'Erfolge', shorten_size: 60)
+        = td_truncate_content_modal(record.future, 'Zukunft', shorten_size: 60)
+        = td_truncate_content_modal(record.comments, t_attr(:comments, Feedback), shorten_size: 60)
+        td= t(record.conversation)
+        td.index-action-cell
+          span= "#{(Time.zone.today - record.created_at.to_date).to_i} Tage"
+          - if record.reviewer.present?
+            span
+              = "Angeschaut von: "
+              = link_to record.reviewer.email, record.reviewer
+        td
+          - if record.reviewer.present?
+            = link_to 'Anzeigen', polymorphic_path([record.volunteer, record.feedbackable_link_target, record]), class: 'index-action-link'
+          - else
+            = link_to 'Angeschaut', polymorphic_path([record.volunteer, record.feedbackable_link_target, record], action: :mark_as_done, q: search_parameters), { method: :put, class: 'btn btn-success btn-xs m-t-10' }
```

Comment:
`btn-success` should be replaced with `btn-default`. They don't want colors in buttons except for the blue Journal buttons. (Changes regarding all other buttons are already merged.)